tasks/apply-mapping/tests/test-apply-mapping-content-gateway-disabled.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-apply-mapping-content-gateway-disabled
+spec:
+  description: |
+    Run the apply-mapping task with a snapshot.spec json and no default configuration for content gateway and no
+    configuration data for any of the components.
```

Comment:
Can you add what the expected outcome is for this scenario? Probably that the resulting snapshot spec still does not contain any content gateway config for the components?

Reply:
added